lib/licensee/project_files/license_file.rb

```diff
@@ -26,6 +30,7 @@ class LicenseFile < Licensee::Project::File
         /\A#{COPYING_REGEX}#{ANY_EXT_REGEX}\z/       => 0.5, # COPYING.textile
         /#{LICENSE_REGEX}/                           => 0.4, # LICENSE-MIT
         /#{COPYING_REGEX}/                           => 0.3, # COPYING-MIT
+        /#{OFL_REGEX}/                               => 0.5, # OFL
```

Comment:
To match the other standards, I believe we'll want `/OFL(#{PREFERRED_EXT_REGEX}` and `/OFL#{ANY_EXT_REGEX}/` as their own entries. It should also be scored lower than `COPYING-*`. when both exist.